src/main/java/com/cloudbees/jenkins/plugins/bitbucket/PullRequestSCMHead.java

```diff
@@ -186,7 +186,11 @@ public ChangeRequestCheckoutStrategy getCheckoutStrategy() {
     @NonNull
     @Override
     public String getOriginName() {
-        return branchName;
+        if (strategy == ChangeRequestCheckoutStrategy.MERGE) {
```

Comment:
No, create a new method. :cry:

Reply:
This does not work for cloud: https://stackoverflow.com/questions/35216431/bitbucket-pull-request-refspec

This line will be broken
https://github.com/jenkinsci/bitbucket-branch-source-plugin/pull/169/files#diff-f47f4b30aafc3302f647c3dbcec3ac48L159